run-prod.sh

```
@@ -0,0 +1 @@
+docker compose -f docker-compose.yaml -f docker-compose.prod.yaml up --build
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)